DigitalEcosystem/publication/bandgap2DMaterials/tpot_autogenerated_pipeline.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
```

Comment:
(1) https://github.com/Exabyte-io/Scientific-Projects/pull/28/files#r724591915
(2) Duplicate code again - can we reuse some of this???

Reply:
None of the code in this file can be re-used or modified by us, it is all automatically generated.

Reply:
https://github.com/Exabyte-io/Scientific-Projects/pull/28/files#r724616133